recipes/iptkl/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "iptkl" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 340dc6265e64a3341f0db05151edf251e1e6c8476401d2a10194fbbaeb16a7db
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
I think this should be: `noarch: python`?

Reply:
Thanks a lot for pointing this out, I am not quite sure, why it was marked as generic instead of python, I used Conda skeleton to create the yaml files. I have changed the files accordingly.